project/Dependencies.scala

```diff
@@ -2,13 +2,13 @@ import sbt._
 
 object Dependencies {
 
-  val scalatest        = "org.scalatest"       %% "scalatest"      % "3.2.3"
-  val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"    % "3.0.1"
-  val smetrics         = "com.evolutiongaming" %% "smetrics"       % "1.0.1"
-  val `kind-projector` = "org.typelevel"        % "kind-projector" % "0.11.0"
+  val scalatest        = "org.scalatest"       %% "scalatest"      % "3.2.13"
+  val `cats-helper`    = "com.evolutiongaming" %% "cats-helper"    % "3.1.1"
+  val smetrics         = "com.evolutiongaming" %% "smetrics"       % "1.0.5-SNAPSHOT"
```

Comment:
please release only :)

Reply:
sure thing, I wanted to update it as soon as I release smetrics :)